windows-find-tools/README.md

```diff
@@ -0,0 +1,23 @@
+# windows-find-tools 
```

Comment:
As [per discussion on Zulip](https://rust-lang.zulipchat.com/#narrow/channel/351149-t-libs.2Fcrates/topic/interior.20decorating.20with.20cc-rs/near/536114787), please state that this is an "internal use" crate.

Reply:
Marked it as internal use